app/controllers/admin/financial_transaction_classes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Admin::FinancialTransactionClassesController < Admin::BaseController
+  inherit_resources
+
+  def new
+    @financial_transaction_class = FinancialTransactionClass.new(params[:financial_transaction_class])
+    render :layout => false
+  end
+
+  def create
+    @financial_transaction_class = FinancialTransactionClass.new(params[:financial_transaction_class])
+    if @financial_transaction_class.valid? && @financial_transaction_class.save
```

Comment:
I think that [`save`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save) already checks for validity, and returns that.